tests/runtime/engine/runner.py

```diff
@@ -137,6 +139,29 @@ def __get_bpffeature():
         bpffeature["jiffies64"] = output.find("jiffies64: yes") != -1
         return bpffeature
 
+    
+    @staticmethod
+    def __wait_for_children(parent_pid, timeout, ps_format, condition):
+        with open(os.devnull, 'w') as dn:
```

Comment:
Unused?

Reply:
I stole this from the original code. TBH, I'm not sure what it's doing but I was afraid to just remove it.